furnace/context.py

```diff
@@ -29,15 +28,18 @@
 from . import pid1
 from .config import NAMESPACES, HOST_NETWORK_BIND_MOUNTS, BindMount
 from .libc import unshare, setns, CLONE_NEWPID
-from .utils import PathEncoder, BindMountContext
+from .utils import PathEncoder
 
 logger = logging.getLogger(__name__)
 
 
 class ContainerPID1Manager:
-    def __init__(self, root_dir: Path, *, isolate_networking=False):
+    def __init__(self, root_dir: Path, *, bind_mounts=None, isolate_networking=False):
```

Comment:
The order of "isolate_networking, bind_mounts" would be more consistent with the `PID1.__init__`'s and `ContainerContext.__init__`'s signature.